cpp/unreal_plugins/WheeledVehicle/Source/WheeledVehicle/VehicleMovementComponent.cpp

```diff
@@ -0,0 +1,62 @@
+//
+// Copyright(c) 2022 Intel. Licensed under the MIT License <http://opensource.org/licenses/MIT>.
+//
+
+#include "WheeledVehicle/VehicleMovementComponent.h"
+
+#include <Eigen/Dense>
+
+#include "CoreUtils/Config.h"
+#include "CoreUtils/Log.h"
+#include "WheeledVehicle/VehicleWheel.h"
+
+UVehicleMovementComponent::UVehicleMovementComponent()
+{
+    SP_LOG_CURRENT_FUNCTION();
+
+    if (!Config::s_initialized_) {
+        return;
+    }
+
+    // setup wheels
+    WheelSetups.SetNum(4);
+
+    UClass* wheel_class = UVehicleWheel::StaticClass();
+
+    WheelSetups[0].WheelClass = wheel_class;
+    WheelSetups[0].BoneName = FName("FL");
+    WheelSetups[0].AdditionalOffset = FVector::ZeroVector; // Offset the wheel from the bone's location
+
+    WheelSetups[1].WheelClass = wheel_class;
+    WheelSetups[1].BoneName = FName("FR");
+    WheelSetups[1].AdditionalOffset = FVector::ZeroVector; // Offset the wheel from the bone's location
+
+    WheelSetups[2].WheelClass = wheel_class;
+    WheelSetups[2].BoneName = FName("RL");
+    WheelSetups[2].AdditionalOffset = FVector::ZeroVector; // Offset the wheel from the bone's location
+
+    WheelSetups[3].WheelClass = wheel_class;
+    WheelSetups[3].BoneName = FName("RR");
+    WheelSetups[3].AdditionalOffset = FVector::ZeroVector; // Offset the wheel from the bone's location
+
+    ChassisWidth       = Config::get<float>("WHEELED_VEHICLE.VEHICLE_COMPONENT.CHASSIS_WIDTH");
+    ChassisHeight      = Config::get<float>("WHEELED_VEHICLE.VEHICLE_COMPONENT.CHASSIS_HEIGHT");
+    DragCoefficient    = Config::get<float>("WHEELED_VEHICLE.VEHICLE_COMPONENT.DRAG_COEFFICIENT");
+    Mass               = Config::get<float>("WHEELED_VEHICLE.VEHICLE_COMPONENT.MASS");
+    InertiaTensorScale = FVector::OneVector;    
```

Comment:
Don't need to set `InertiaTensorScale` because this is the default value.

Reply:
Yup, removed it